src/interfaces.ts

```diff
@@ -15,6 +15,7 @@ export interface IUsers {
     complement: string
     quota: UserQuota
     validatedMail: boolean
+    customerId: string
```

Comment:
esse campo tá ambíguo.. oq é customerID? 

se for um campo referente a pagamento, usar algo como `paymentServiceCustomerId`

Reply:
Concordo com o Marcus. É bom deixar claro que é um id referente ao serviço de um terceiro